### PR TITLE
Add email messages spec

### PR DESCRIPTION
**Pull request by vova07**
Add `Email Message` definition and related paths.

### CODE REVIEW

File: spec/definitions/EmailNotifications/EmailMessage.yaml [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  eventType:
+    description: The email notification's event
+    type: string
+    enum:
+      - subscription-activated
+      - subscription-canceled
+  sender:
```
Comment (vova07) [inline spec/definitions/EmailNotifications/EmailMessage.yaml]: Should we follow the [`RFC 5322`](https://tools.ietf.org/html/rfc5322#section-3.6.2) and call this attribute instead as`from`?

File: spec/definitions/EmailNotifications/EmailMessage.yaml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    description: The email message's sender address list
+    type: array
+    minItems: 1
+    items:
+      type: string
+  recipients:
```
Comment (vova07) [inline spec/definitions/EmailNotifications/EmailMessage.yaml]: Should we follow the [`RFC 5322`](https://tools.ietf.org/html/rfc5322#section-3.6.3) and call this attribute instead as`to`?

File: spec/definitions/EmailNotifications/EmailMessage.yaml (hunk 1 above)
Comment (vova07) [reply spec/definitions/EmailNotifications/EmailMessage.yaml]: As per same RFC: should we follow it and support multiple `from` addresses?

Comment (adamaltman) [reply spec/definitions/EmailNotifications/EmailMessage.yaml]: Yes `from`.  No multiple.

File: spec/definitions/EmailNotifications/EmailMessage.yaml [hunk 3]
```diff
@@ -0,0 +1,78 @@
...
+  - subject
+  - text
+  - html
+properties:
+  id:
+    description: The email message's identifier string
+    readOnly: true
+    allOf:
+      - $ref: "#/definitions/ResourceId"
+  eventType:
```
Comment (adamaltman) [inline spec/definitions/EmailNotifications/EmailMessage.yaml]: remove, right?

Comment (vova07) [reply spec/definitions/EmailNotifications/EmailMessage.yaml]: Yes

**Comment by vova07** [comment]
please review

**Comment by vova07** [comment]
please review

**Pull request closed without merging**